sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsRequestOptions.cs

```diff
@@ -39,7 +39,7 @@ internal TextAnalyticsRequestOptions(bool includeStatistics, string modelVersion
         /// model versions, see operation-specific documentation, for example:
         /// <a href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/how-tos/text-analytics-how-to-sentiment-analysis#model-versioning"/>.
         /// </summary>
-        public string ModelVersion { get; set; }
+        public string ModelVersion { get; set; } = "latest";
```

Comment:
does the Analyze endpoint still fails if no model version is passed? If so, I will recommend to set it only on those conditions and not for all the library.